Tools/sitl_run.sh

```diff
@@ -74,6 +74,9 @@ if [ "$model" == "" ] || [ "$model" == "none" ]; then
 	if [ "$program" == "jsbsim" ]; then
 		echo "empty model, setting rascal as default for jsbsim"
 		model="rascal"
+	elif [ "$program" == "sih_sim" ]; then
```

Comment:
More of a nitpick, but could we get rid of the underscore? I think this would get confusing with our `make` target distinguishing models/worlds with underscores